robocopkaka/bootcamp-29-project
# #157672162 View events in a center

#### What does this PR do?
allow users view events belonging to a center on the center's page
#### Description of Task to be completed?
* write a method that consumes the endpoint to retrieve events belonging to a center
* write the action and action creator for retrieving events belonging to a center
* add action types relevant to retrieving events belonging to a center
* modify initial state to include a center object inside the centers object
* add reducer methods for retrieving events belonging to a center
* modify the CenterDetail component
* add relevant css
* call the action responsible for retrieving events in a center inside the SingleCenter component
#### How should this be manually tested?
* clone this repo
* cd into the project folder
* run `npm install` to install all dependecies
* run `npm run start:all` to start up the application
* go to `localhost:8080` on your browser
* click on `Signup`
* enter your details
* click on `Signup`
* request for admin access
* click on `Centers` on the navbar
* click on the red add button at the bottom right corner
* enter the details of the center
* click on `Add Center`
* add at least ten more centers
* click on `Centers`
* select a center
* click on the red add button at the bottom right corner
* enter the details of the event
* click on `Add Event`
* you should see the newly created event on the center's page
#### Any background context you want to provide?
> this part of the app was broken after I removed the events array from the object that was returned when a user wanted to get the details of a single center
#### What are the relevant pivotal tracker stories?(if applicable)
#157672162
